Gems/Atom/Feature/Common/Assets/Passes/RayTracingAccelerationStructure.pass

```diff
@@ -11,6 +11,12 @@
                     "Name": "RayTracingTLAS",
                     "SlotType": "Output",
                     "ScopeAttachmentUsage": "Shader"
+                },
+                {
+                    "Name": "SkinnedMeshes",
+                    "ShaderInputName": "NoBind",
+                    "SlotType": "Input",
+                    "ScopeAttachmentUsage": "Shader"
                 }
```

Comment:
There is a slot+connection here, you also search for the buffer in the C++ class. Either use the slot when calling `framegraph.useAttachment` or remove it entirely.

Reply:
This was not resolved. You can either remove the slot+connections, or use the slot in C++. Currently the slot is not used, and changing the connection won't do anything.